### ISSUE TITLE
Inconsistent sizing of studio thumbnails

### ISSUE THREAD

**Issue by mxmou** [LLK/scratch-www#5598]
There are currently many different aspect ratios of studio thumbnails used in different places on the website:
- 5 : 4 (My Stuff)
- 4 : 3 (new studio page)
- 20 : 13 (old studio page)
- 17 : 10 (everywhere else - front page, Explore, search, profile pages)

On the new studio page, the thumbnail is cropped to the 4 : 3 aspect ratio. On other pages, it's scaled. Because of these inconsistencies, it's very hard to make a studio thumbnail which looks good everywhere.

One of the aspect ratios (probably 4 : 3 or 17 : 10) and resizing behavior should be chosen and used everywhere.

**Comment by BryceLTaylor** [comment]
@mxmou you are totally right.  If you're interested, we would love some help identifying all of the aspect ratios and sizes of thumbnails and avatars that we use throughout the site so we can address this holistically.

**Comment by mxmou** [comment]
The exact sizes used for various thumbnails I could find are:

### Project thumbnails
* front page: loaded as 480x360, displayed as 144x108
* Starter Projects (scratch-www), old studio page, profile page (scratchr2): loaded and displayed as 144x108
* Explore, search, project page (Remixes - needs to match width of studio thumbnails below): loaded as 480x360, displayed as 204x152
* new studio page: loaded as 480x360, displayed as 265x199
* Annual Report: loaded as 155x108, displayed as 204x152 (this makes them appear blurred, which is probably not intended)
* My Stuff (scratchr2): loaded as 100x75, displayed as 100x80 (incorrect aspect ratio - 5 : 4 instead of 4 : 3 like everywhere else)

### Studio thumbnails
* front page: resolution as uploaded, displayed as 170x100 (aspect ratio 17 : 10)
* Explore, search, project page (Studios - needs to match width of studio thumbnails above): loaded as 170x100, displayed as 204x120 (blurred; aspect ratio 17 : 10)
* new studio page: loaded as 170x100, displayed as 296x191 (even more blurred; aspect ratio ≈ 20 : 13)
* old studio page (scratchr2): loaded and displayed as 200 : 130 (aspect ratio 20 : 13)
* profile page (scratchr2): loaded and displayed as 170 : 100 (aspect ratio 17 : 10)
* My Stuff (scratchr2): resolution as uploaded, displayed as 100x80 (aspect ratio 5 : 4)

I think the aspect ratio was 4 : 3 on the Scratch 1.x website.

### User avatars
* navigation bar: loaded as 32x32, displayed as 24x24
* front page (What's Happening?): resolution as uploaded, displayed as 32x32
* Explore, search, project page (Remixes), Annual Report (project cards): loaded and displayed as 32x32
* project page (creator thumbnail): loaded and displayed as 48x48
* project & new studio page (next to comment compose box): loaded as 32x32, displayed as 48x48 (blurred)
* project & new studio page (comments): loaded as 60x60, displayed as 48x48
* Messages (comments): loaded as 32x32, displayed as 40x40 (blurred)
* new studio page (projects, curators): loaded as 90x90, displayed as 42x42, incorrect resizing - cover instead of stretch
* Annual Report (quotes): loaded as 80x80, displayed as 60x60
* Our Team, Annual Report (Scratch Team): loaded and displayed as 80x80
* profile page (header) (scratchr2): loaded as 60x60, displayed as 55x55
* profile & old studio page (comments, including the avatar next to the compose box) (scratchr2): loaded as 60x60, displayed as 45x45
* profile page (following & followers), old studio page (curators) (scracthr2): loaded and displayed as 60x60
* forums (scratchr2): loaded and displayed as 90x90

**Comment by BryceLTaylor** [comment]
@mxmou Thanks so much!  That's will really help some of our upcoming work!
